Modules/vc-module-orders/VirtoCommerce.OrdersModule.Web/Scripts/services/showPriceFilter.js

```diff
@@ -0,0 +1,13 @@
+angular.module('virtoCommerce.orderModule')
```

Comment:
Need to check if this filter would be needed when prices will be null instead of zeroes.

Reply:
without filter money fields are blank. I think it's not good.